frontend/src/metabase/components/Popover/TippyPopover.unit.spec.js

```diff
@@ -52,22 +50,6 @@ describe("Popover", () => {
     );
   });
 
-  it("should respect the z-index theme setting", async () => {
-    render(
-      <ThemeProvider theme={{ other: { popover: { zIndex: 505 } } }}>
-        <TippyPopover content={<Content />} visible>
-          {defaultTarget}
-        </TippyPopover>
-      </ThemeProvider>,
-    );
-
-    // Tippy's root element is a direct parent, but it has no identifiable role.
-    // eslint-disable-next-line testing-library/no-node-access
-    expect(screen.getByRole("tooltip").parentElement).toHaveStyle({
-      zIndex: 505,
-    });
-  });
```

Comment:
We don't need this test anymore as the same test has been added on Cypress. The problem is that Jest does not support CSS variables propagation in their CSSOM, so using `getComputedStyle` here would just return the CSS variable string `var(--mb-overlay-z-index)` instead of `505`